misk-crypto/src/main/kotlin/misk/crypto/ExternalDataKeys.kt

```diff
@@ -0,0 +1,18 @@
+package misk.crypto
+
+import javax.inject.Qualifier
+
+/**
+ * This annotation is used to decorate the collection of external data keys used in the service.
+ *
+ * External data keys are defined in the configuration as a [Map]<[KeyAlias], [KeyType]>.
+ *
+ * This annotation can be useful for cases where one would need access to all external data
+ * keys available to the service.
+ */
+@Qualifier
+@Target(
+  AnnotationTarget.FIELD,
+  AnnotationTarget.VALUE_PARAMETER
+)
+annotation class ExternalDataKeys()
```

Comment:
Should this be `public` or `internal`? I assume we'd export primitives/key managers, not raw keys.

Reply:
This is a first step towards exposing a more convenient interface and for now, I just want to be able to expose the map of key name -> key type for external keys.

I think it's useful to have it as a public annotation so that other consumers could use it as well.
It doesn't expose any key material so it should be safe to do.

In other words - this is just a convenient way to get to the external keys block specified in the configuration.